test/pending-trip.js

```diff
@@ -16,10 +16,11 @@ describe('pending-trip.js', function() {
                 destination: "Stockholm, Sweden"
             }]);
         });
-    });
-    it('should return a message if the traveler has no pending trips', function() {
-        const noPendingTrips = findPendingTrips(4, trips, destinations)
+    
+        it('should return a message if the traveler has no pending trips', function() {
```

Comment:
I think this was very thoughtful and informational instead of just returning a boolean and leaving a blank display!